pkg/convert/pot_test.go

```diff
@@ -560,6 +563,98 @@ func TestFromPot(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "fuzzy param before empty id",
+			input: []byte(`#, fuzzy
+							msgid ""
+							msgstr ""
+							"Language: en\n"
+							#. a greeting
+							msgid "Hello"
+							msgstr "Hello, world!"
+
+							#. a farewell
+							msgid "Goodbye"
+							msgstr "Goodbye, world!"
+			`),
+			expected: model.Messages{
+				Language: language.English,
+				Messages: []model.Message{
+					{
+						ID:          "Hello",
+						Message:     "Hello, world!",
+						Description: "a greeting",
+						Fuzzy:       true,
+					},
+					{
+						ID:          "Goodbye",
+						Message:     "Goodbye, world!",
+						Description: "a farewell",
+						Fuzzy:       false,
+					},
+				},
+			},
+		},
+		{
+			name: "msgid and msgstr empty headers",
+			input: []byte(`#, fuzzy
+							"Language: en\n"
```

Comment:
This does not look right. Should it be?

```suggestion
							Language: en
```

Reply:
In all examples, that I found Language header ends with "\n", so I thought it should be mandatory.

https://www.drupal.org/community/contributor-guide/reference-information/localize-drupal-org/working-with-offline/po-and-pot-files
https://gist.github.com/bosskovic/5930785
https://www.simultrans.com/blog/what-are-.po-files